build/build.ps1

```diff
@@ -80,14 +80,15 @@ function LocateVisualStudio {
   $vswhereVersion = GetVersion("VSWhereVersion")
   $vsWhereDir = Join-Path $ToolsRoot "vswhere\$vswhereVersion"
   $vsWhereExe = Join-Path $vsWhereDir "vswhere.exe"
-  
+
   if (!(Test-Path $vsWhereExe)) {
     Create-Directory $vsWhereDir
     Write-Host "Downloading vswhere"
+    [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
```

Comment:
nit: This should be at the top of the code, so it impacts all downloads (existing and future) rather than just the VSWhere download